src/Common/FitOnFhir.Common/Handlers/RequestHandlerBase.cs

```diff
@@ -0,0 +1,57 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.Health.Common.Handler;
+using Microsoft.Health.FitOnFhir.Common.Requests;
+
+namespace Microsoft.Health.FitOnFhir.Common.Handlers
```

Comment:
It would be a good idea to have a `RequestHandlerBaseTests` class that all other handlers can derive from. The initial route validation could be handled there instead of each test suite needing to validate route validation.

e.g

**GivenRequestRouteIsNull_WhenEvaluateCalled_NullReturned**
**GivenRequestRouteIsNotContainedInHanldedRoutes_WhenEvaluateCalled_NullReturned**